### ISSUE TITLE
When update to spring-boot 2.0, armeria-spring-boot-start has compile error.

### ISSUE THREAD

**Issue by eonezhang** [line/armeria#1048]
when I update my app to spring-boot 2.0, my app cannot startup due to io.micrometer:micrometer-spring-legacy dependency.

here is my build.gradle 
```
dependencies {
    compile 'com.linecorp.armeria:armeria-spring-boot-starter'
    compile 'com.linecorp.armeria:armeria-thrift'
    compile 'org.springframework.boot:spring-boot-starter'
    compile 'org.springframework.boot:spring-boot-starter-web'
    compile 'org.aspectj:aspectjweaver'
    compile 'org.springframework:spring-aop'
}
```

**Comment by trustin** [comment]
We never tested our spring-boot integration with spring-boot 2.0 so far. What happens if you exclude `micrometer-spring-legacy` from transitive dependencies?

**Comment by eonezhang** [comment]
it seems OK when exclude the dependency.
```
com.linecorp.armeria:
  armeria-thrift: {version: &ARMERIA_VERSION '0.58.1'}
  armeria-zookeeper: {version: *ARMERIA_VERSION}
  armeria-spring-boot-starter:
    version: *ARMERIA_VERSION
    exclusions:
    - io.micrometer:micrometer-spring-legacy
```

**Comment by trustin** [comment]
@eonezhang Great! @imasahiro is working on this at #1049. Please feel free to review.

**Comment by eonezhang** [comment]
I'm following this, @trustin @imasahiro  thanks for your quick response.

**Comment by imasahiro** [comment]
#1049 was merged. Could you try to use `armeria-spring-boot-*` please?

**Comment by trustin** [comment]
Should be fixed via #1049. Please open a new issue if this is still a problem.
